FoodBookApp/UI/Views/CreateReview/CreateReview2View.swift

```diff
@@ -105,6 +105,18 @@ struct CreateReview2View: View {
                         }
                     }.padding().background(customGray2).cornerRadius(10)
                 }.padding(.horizontal).padding(.top)
+                .navigationTitle("Review")
+                .toolbar{
+                    Button(action: {
+                        print("Uploads this review!")
+                    }, label: {
+                        Text("Done")
+                            .frame(maxWidth: .infinity)
+                            .padding()
```

Comment:
Check this padding since it's different from the "Next" in the previous view (see video)

https://github.com/ISIS3510-202410-Team23/SwiftApp/assets/69609680/72bd831c-5089-49b7-b333-3ca4ae3d6769

Reply:
I think the padding we can leave to as a hotfix later.